todo_app/data/classes/test_view_model.py

```diff
@@ -0,0 +1,48 @@
+from todo_app.data.classes.Item import Item
+from todo_app.data.classes.ViewModel import ViewModel
+
+def test_view_model_not_started_items_property_returns_items_with_status_not_started():
+    # Arrange
+    view_model_and_filtered_items = create_view_model_with_not_started_and_done_items()
+    view_model = view_model_and_filtered_items['view_model']
+    not_started_items = view_model_and_filtered_items['not_started_items']
+
+    # Act 
+    result = view_model.not_started_items
+
+    # Assert
+    assert result == not_started_items
+
+def test_view_model_done_items_property_returns_items_with_status_done():
+    # Arrange
+    view_model_and_filtered_items = create_view_model_with_not_started_and_done_items()
+    view_model = view_model_and_filtered_items['view_model']
+    done_items = view_model_and_filtered_items['done_items']
+
+    # Act 
+    result = view_model.done_items
+
+    # Assert
+    assert result == done_items
+
+def create_view_model_with_not_started_and_done_items():
```

Comment:
Nothing wrong with this approach (I can't think of any particular pros or cons), but worth noting that pytest fixtures are a common way of achieving this with a slightly different syntax:
```python
@pytest.fixture
def view_model_and_filtered_items():
    ...
    return ...
```


And that gets injected when requested as a test argument:
```python
def test_view_model_not_started_items_property_returns_items_with_status_not_started(view_model_and_filtered_items):
    ...
```

Reply:
Ah of course, I think that this is a nicer approach, so I've made this change.